docker_tippecanoe/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:18.04
+#based on https://hub.docker.com/r/klokantech/tippecanoe
+#with newer ubuntu
+
+ARG BUILD_DATE
+
+LABEL maintainer="wwatkins@usgs.gov"
+LABEL version=0.1.0
+LABEL org.label-schema.build-date=$BUILD_DATE
+
+# Update repos and install dependencies
+RUN apt-get update \
```

Comment:
Would also suggest adding `apt-get clean && apt-get autoremove` at the end of these updates